File: Model_Day_4/RiskAssessor.py
class RiskAssessor:
    def assess(self,customer):
        if customer.age < 21:
            return "Not Eligible"
        elif customer.salary >= 8000000:
            return "Eligible"
        elif customer.salary >= 3000000:
            return "Manual Review"
        else:
            return "Not Eligible"

File: Model_Day_4/RiskAssessor.java
package Model_Day_4;

public class RiskAssessor{
    public String assess(Customer2 customer){
        String decision;
        if(customer.age < 21){
            decision = "Not Eligible";
        } else if (customer.salary >= 8000000){
            decision = "Eligible";
        } else if (customer.salary >= 3000000){
            decision = "Manual Review";
        } else {
            decision = "Not Eligible";
        }
        return decision;
    }
}